### ISSUE TITLE
Build Failing on Android

### ISSUE THREAD

**Issue by karthi238** [MadeinFrance/react-native-share-pdf#52]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-share-pdf@0.0.6` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-share-pdf/android/src/main/java/com/maximegerbe/reactnative/sharefile/ShareFile.java b/node_modules/react-native-share-pdf/android/src/main/java/com/maximegerbe/reactnative/sharefile/ShareFile.java
index 8667de9..520fc6e 100644
--- a/node_modules/react-native-share-pdf/android/src/main/java/com/maximegerbe/reactnative/sharefile/ShareFile.java
+++ b/node_modules/react-native-share-pdf/android/src/main/java/com/maximegerbe/reactnative/sharefile/ShareFile.java
@@ -8,7 +8,7 @@ package com.maximegerbe.reactnative.sharefile;
 
 import android.content.Intent;
 import android.net.Uri;
-import android.support.v4.content.FileProvider;
+import androidx.core.content.FileProvider;
 import android.util.Base64;
 import android.app.Activity;
 
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by artureg** [comment]
@MadeinFrance please update your library

**Comment by MadeinFrance** [comment]
Should be fixed in [v0.0.7](https://www.npmjs.com/package/react-native-share-pdf/v/0.0.7)

**Comment by artureg** [comment]
thanks
